learn/getting_started/search_preview.mdx

```diff
@@ -0,0 +1,11 @@
+## Search preview
```

Comment:
We have a page on Search preview under "What is Meilisearch": https://www.meilisearch.com/docs/learn/what_is_meilisearch/search_preview

It contains the same information as this page but also mentions the `--http-addr` and `--env` CLI options.